triton-vm/src/table/master_table.rs

```diff
@@ -583,6 +706,12 @@ impl MasterTable for MasterMainTable {
             Some(table.view())
         }
     }
+
+    fn trace_randomizer_for_column(&self, idx: usize) -> Polynomial<Self::Field> {
+        let mut rng = StdRng::from_seed(self.column_randomizer_seeds[idx]);
+        let coefficients = (0..self.num_trace_randomizers).map(|_| rng.gen()).collect();
+        Polynomial::new(coefficients)
+    }
```

Comment:
@aszepieniec: I don't think a full review from you is required:[^review] given that this PR does not change the verifier at all, I am quite certain that Triton VM remains as complete and sound as it was before. The critical question is that of zero-knowledge, which should be confined to this highlighted method.[^range] Could you have a look at that?

[^review]: if you disagree or feel like reviewing the changes, please go ahead anyway
[^range]: For example, should the range be `0..=self.num_trace_randomizers`?

Reply:
The previous implementation was erring on the side of caution. c2618788295328ae2ba311ff512d2e73a7216219 suggests an alternative approach. What do you think of that?